src/cljs/org/broadinstitute/firecloud_ui/page/workspace/monitor/workflow_details.cljs

```diff
@@ -0,0 +1,119 @@
+(ns org.broadinstitute.firecloud-ui.page.workspace.monitor.workflow-details
+  (:require
+    [dmohs.react :as react]
+    [org.broadinstitute.firecloud-ui.common :as common]
+    [org.broadinstitute.firecloud-ui.common.components :as comps]
+    [org.broadinstitute.firecloud-ui.common.icons :as icons]
+    [org.broadinstitute.firecloud-ui.common.style :as style]
+    [org.broadinstitute.firecloud-ui.endpoints :as endpoints]
+    [org.broadinstitute.firecloud-ui.page.workspace.monitor.common :as moncommon]
+    [org.broadinstitute.firecloud-ui.utils :as utils]
+    ))
+
+
+(defn- create-field [label & contents]
```

Comment:
This seems like another pretty generic method suitable for utils or style. I remember writing one of these for importing method configs.